docs/types.md

```diff
@@ -1045,6 +1045,13 @@ Below are the currently supported file operations:
 | `make-dir` | `mkdir`    | path                   | `mkdir -p`       | Create a directory at the given path. The path make already exist. Any required parent directories will be created. |
 | `remove`   | `rm`       | path                   | `rm -rf`         | Remove the given path. The path may be a file or directory. The path may not exist. |
```

Comment:
What do you think about changing the relevant arguments here from "path" to "glob path" or "glob pattern"?

It may be a little more confusing for those that don't know what globs are, but your next paragraph should clear it up.

I have a comment further down that talks about the choice between "glob pattern" and "glob path".